### PR TITLE
added new Controller and change some fields

### PR DESCRIPTION
**Pull request by Zingerd**

### CODE REVIEW

File: core/src/main/java/greencity/controller/EcoNewsController.java [hunk 1]
```diff
@@ -364,4 +352,15 @@ public void like(@RequestParam("id") Long id, @ApiIgnore @CurrentUser UserVO use
         @ApiIgnore @CurrentUser UserVO user) {
         return ResponseEntity.status(HttpStatus.OK).body(ecoNewsService.checkNewsIsLikedByUser(econewsId, user));
     }
+
+    /**
+     * Method for getting some fields in eco news by id.
+     *
+     * @return dto {@link EcoNewContentSourceDto}.
+     */
+    @GetMapping("/getContentAndSourceForEcoNews/{id}")
```
Comment (VadymMakitra) [inline core/src/main/java/greencity/controller/EcoNewsController.java:361]: Please read https://restfulapi.net/

File: service-api/src/main/java/greencity/dto/econews/EcoNewContentSourceDto.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+import lombok.*;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@ToString
+@Builder
+@EqualsAndHashCode
```
Comment (VadymMakitra) [inline service-api/src/main/java/greencity/dto/econews/EcoNewContentSourceDto.java:13]: It is not good practise to use Builder and Setter

File: service/src/main/java/greencity/service/EcoNewsServiceImpl.java [hunk 3]
```diff
@@ -578,12 +579,31 @@ private EcoNewsDto getEcoNewsDto(EcoNews ecoNews, List<String> list) {
...
+
+    /**
+     * Method for getting some fields in eco news by id.
+     *
+     * @param id - {@link Long} eco news id.
+     * @return dto {@link EcoNewContentSourceDto}.
+     */
+    public EcoNewContentSourceDto findEcoNewContentSourceDtoById(Long id) {
+        EcoNews ecoNews = ecoNewsRepo.findById(id)
+            .orElseThrow(() -> new NotFoundException(ErrorMessage.ECO_NEWS_NOT_FOUND_BY_ID + id));
```
Comment (VadymMakitra) [inline service/src/main/java/greencity/service/EcoNewsServiceImpl.java:599]: Please verify Bad Request in cotroller test's;

**Review by VadymMakitra — CHANGES_REQUESTED** [review]

**Comment by VadymMakitra** [comment]
Resolve comment

**Review by thesvider — CHANGES_REQUESTED** [review]
Please resolve comments

**Review by Max877866 — APPROVED** [review]

**Review by VadymMakitra — APPROVED** [review]

**Review by thesvider — APPROVED** [review]

**Pull request merged**